src/pages/index.js

```diff
@@ -1,95 +1,245 @@
+// ! ||--------------------------------------------------------------------------------||
+
+api
+  .getUserInfo()
+  .then((userData) => {
+    userInfo.setUserInfo({
+      title: userData.name,
+      description: userData.about,
+    });
+    userInfo.setAvatar(userData.avatar);
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+api
+  .getInitialCards()
+  .then((cards) => {
+    cardsList.renderItems(cards);
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
 // // ! ||--------------------------------------------------------------------------------||
 // // ! ||                                 Event Handlers                                 ||
 // // ! ||--------------------------------------------------------------------------------||
 
+function handleDeleteClick(card) {
+  const id = card.getID();
+  deleteCardModal.open(() => {
+    deleteCardModal.setLoading();
+    api
+      .removeCard(id)
+      .then(() => {
+        card.handleDeleteCard();
+        deleteCardModal.close();
+      })
+      .catch((error) => {
+        console.error("Error removing card:", error);
+      })
+      .finally(() => {
+        deleteCardModal.resetButtonText();
+      });
+  });
+}
+
+function handleAddLike(card) {
+  const id = card.getID();
+  api
+    .addLike(id)
+    .then((updatedCardData) => {
+      card.setLiked(true);
+    })
+    .catch((err) => {
+      console.error(err);
+    });
+}
+
+function handleRemoveLike(card) {
+  const id = card.getID();
+  api
+    .removeLike(id)
+    .then((updatedCardData) => {
+      card.setLiked(false);
+    })
+    .catch((err) => {
+      console.error(err);
+    });
+}
 function handleAddCardFormSubmit(inputValues) {
-  console.log(this._modalForm);
   const name = inputValues.title;
   const link = inputValues.url;
-  const newCardElement = createCard({ name, link });
-  cardsList.prependItem(newCardElement);
-  addCardModal.close();
-  addCardFormValidator.resetValidation();
-  addCardModal.reset();
+  addCardModal.setLoading();
+  api
+    .addCard({ name, link })
+    .then((newCardData) => {
+      const newCardElement = createCard(newCardData);
+      cardsList.prependItem(newCardElement);
+      addCardModal.close();
+      addCardFormValidator.resetValidation();
+      addCardModal.reset();
+    })
+    .catch((err) => {
+      console.error(err);
+    })
+    .finally(() => {
+      addCardModal.resetButton();
+    });
+}
+
+function handleUpdateAvatarFormSubmit(inputValues) {
+  const link = inputValues.url;
+  updateAvatarModal.setLoading();
+  api
+    .changeProfilePicture(link)
+    .then((data) => {
+      userInfo.setAvatar(data.avatar);
+      updateAvatarModal.reset();
+      updateAvatarModal.close();
+    })
+    .catch((err) => {
+      console.error(err);
+    })
+    .finally(() => {
+      updateAvatarModal.resetButton();
+    });
+}
+
+function handleUpdateAvatar() {
+  const link = document.getElementById("avatar-image-url").value;
```

Comment:
Only method `_getInputValues` collects all the data of the inputs **inside** `PopupWithForm` and passes it to the call of the `submit handler`. So you need to use the collected data rather than the inputs.